src/index.js

```diff
@@ -41,11 +41,11 @@ export function isValid(value) {
     return false;
   }
 
-  return isValidSsn(value) || isValidItin(value) || isValidEin(value);
+  return isValidSsn(sanitizedValue) || isValidItin(sanitizedValue) || isValidEin(sanitizedValue);
```

Comment:
I noticed that the regex applied to the sanitized value is different form the `sanitize` function 🙈 

```
export function sanitize(value) {
  return String(value).replace(/\D+/g, '');
}
```

Reply:
However, it is the same as:

https://github.com/uphold/ssn-validator/blob/master/src/index.js#L32

Ok the other two [ein](https://github.com/uphold/ein-validator/blob/master/src/index.js#L53) and [itin](https://github.com/uphold/itin-validator/blob/master/src/index.js#L25) don't have it